SmartDeviceLink/public/SDLLockScreenViewController.m

```diff
@@ -87,6 +92,18 @@ - (void)setLockedLabelText:(NSString *_Nullable)lockedLabelText {
     [self sdl_layoutViews];
 }
 
+- (void)setModalPresentationStyle:(UIModalPresentationStyle)modalPresentationStyle {
+    if (@available(iOS 13.0, *)) {
+        if (modalPresentationStyle == UIModalPresentationAutomatic) {
+            _modalPresentationStyle = UIModalPresentationFullScreen;
+        } else {
+            _modalPresentationStyle = modalPresentationStyle;
+        }
+    } else {
+        _modalPresentationStyle = modalPresentationStyle;
+    }
+}
+
```

Comment:
This seems unnecessary to me. We just override it to be `UIModalPresentationFullScreen`. The dev should be subclassing this and can override it if they want something different.

```suggestion
```